core/src/main/scala/org/http4s/multipart/MultipartDecoder.scala

```diff
@@ -31,17 +32,17 @@ private[http4s] object MultipartDecoder {
     }
 
 
-  def gatherParts(h: Handle[Task, Either[Headers, Byte]]): Pull[Task, Part, Either[Headers, Byte]] = {
-    def go(part: Part, lastWasLeft: Boolean)(h: Handle[Task, Either[Headers, Byte]]): Pull[Task, Part, Either[Headers, Byte]] = {
+  def gatherParts(h: Handle[Task, Either[Headers, ByteVector]]): Pull[Task, Part, Either[Headers, ByteVector]] = {
+    def go(part: Part, lastWasLeft: Boolean)(h: Handle[Task, Either[Headers, ByteVector]]): Pull[Task, Part, Either[Headers, ByteVector]] = {
       h.receive1Option {
         case Some((Left(headers), h1)) =>
           if (lastWasLeft){
             go(Part(Headers(part.headers.toList ::: headers.toList), EmptyBody), true)(h1)
           } else {
            Pull.output1(part) >> go(Part(headers, EmptyBody), true)(h1)
           }
-        case Some((Right(byte), h1)) =>
-          go(part.copy(body = part.body.append(Stream.emit[Task, Byte](byte))), false)(h1)
+        case Some((Right(bv), h1)) =>
+          go(part.copy(body = part.body.append(Stream.chunk(ByteVectorChunk(bv)))), false)(h1)
```

Comment:
Finally we use `ByteVectorChunk`

Reply:
I was just trying to get rid of it earlier this week!

It's also used in blaze.